WhyNot/WhyNot/UNO-102/PeopleImagesView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  PeopleImagesView.swift
+//  WhyNot
+//
+//  Created by Fernando Santos on 03/11/22.
+//
+
+import SwiftUI
+
+struct PeopleImagesView: View {
+    @ObservedObject var viewModel: PeopleVM
+
+    init(images: [UIImage]) {
+        viewModel = PeopleVM(images: images)
+    }
+
+    var body: some View {
+        HStack(spacing: -Space.halfSpace) {
+            ForEach(0 ..< viewModel.getActualMaxImagesNumber()) { imageIdx in
+                let personImage = viewModel.getPeopleImage(index: imageIdx)
+                PersonImage(personImage: Image(uiImage: personImage))
+            }
```

Comment:
Aqui você poderia usar uma função que já retorna as imagens que devem ser mostradas e iterar sobre elas, aí vc só precisa chamar uma função ao invés de duas :)

```suggestion
            ForEach(viewModel.getImagesToShow()) { image in
                PersonImage(image)
            }
```